fml/message_loop_task_queues.cc

```diff
@@ -12,8 +12,6 @@ namespace fml {
 
 std::mutex MessageLoopTaskQueues::creation_mutex_;
 
-const size_t TaskQueueId::kUnmerged = ULONG_MAX;
```

Comment:
Can we instead change FlutterEngineTest.mm to refer to `FlutterEngine.h` instead of `FlutterEngine_Internal.h`?  We shouldn't be testing against the internal header, and it's not needed for any of the tests in there anyway.